.github/workflows/createNewVersion.yml

```diff
@@ -26,9 +26,21 @@ on:
         required: true
 
 jobs:
+  validateActor:
+    runs-on: ubuntu-latest
+    outputs:
+      HAS_WRITE_ACCESS: ${{ contains(fromJSON('["write", "admin"]'), steps.getUserPermissions.outputs.PERMISSION) }}
+    steps:
+      - name: Get user permissions
+        id: getUserPermissions
+        run: echo "::set-output name=PERMISSION::$(gh api /repos/Expensify/App/collaborators/roryabraham/permission | jq -r '.permission')"
```

Comment:
Can you explain this line and why it is hardcoding your user?

Reply:
Fixed

Reply:
And all it's doing is querying [this API endpoint](https://docs.github.com/en/rest/collaborators/collaborators#get-repository-permissions-for-a-user) and then outputting the `permission` field of the response.

I found in my testing that I had to use a PAT instead of the regular GitHub token, which was surprising because it's just a READ endpoint. I think it must be because it's using `gh api` and it's not validating the actual endpoint used?